Mathlib/Algebra/Hom/Centroid.lean

```diff
@@ -498,7 +492,7 @@ theorem toEnd_int_cast (z : ℤ) : (z : CentroidHom α).toEnd = ↑z :=
   rfl
 #align centroid_hom.to_End_int_cast CentroidHom.toEnd_int_cast
 
-instance : Ring (CentroidHom α) :=
+instance ring : Ring (CentroidHom α) :=
```

Comment:
Shouldn't this be `instRing` or something like that?

Reply:
Oh, I was being consistent with `CentroidHom.commRing`, but that's not an instance. Thanks for pointing that out.